DQMOffline/Trigger/python/HigPhotonJetHLTOfflineSource_cfi.py

```diff
@@ -28,7 +28,7 @@
     dirname = cms.untracked.string("HLT/Higgs/PhotonJet"), 
     verbose = cms.untracked.bool(False), # default: False
     triggerAccept = cms.untracked.bool(True), # default: True 
-    triggerResultsToken = cms.InputTag("TriggerResults"),
+    triggerResultsToken = cms.InputTag("TriggerResults","","HLT"),
```

Comment:
Wait, isn't the example you make already not well-defined in the current version of the framework ?

I mean, assuming the input already contains a collection called "producer", something like
```python
process = cms.Process('AMBIGUOUS')

process.source = ...

process.producer = cms.EDProducer('MyProducer')
process.analyzer = cms.EDANalyzer('MyAnalyzer')

process.path1 = cms.Path( process.analyzer )
process.path2 = cms.Path( process.producer + process.analyzer )
```
would give different results for "analyzer" depending if path1 runs before or after path2...